tests/testutil/matchers/matchers.go

```diff
@@ -121,7 +121,15 @@ func jsonContains(a, e interface{}, path []string) []string {
 			return path
 		}
 	case reflect.String:
-		if a.(string) != e.(string) {
+		_a := a.(string)
+		_e := e.(string)
+		prefix := "REGEXP:"
+		if strings.HasPrefix(_e, prefix) {
+			eReg := regexp.MustCompile(_e[len(prefix):])
+			if !eReg.MatchString(_a) {
+				return path
+			}
+		} else if _a != _e {
```

Comment:
Alternatively - add `jq` as matcher... however it seems to be overkill for this task (And I am not sure that go-implementations supports `match` filter)

https://developer.zendesk.com/documentation/integration-services/developer-guide/jq-cheat-sheet/#extracting-a-pattern-from-a-string